### ISSUE TITLE
Failed to run process: diskutil quiet addPartition disk0s4 %Linux% %noformat% 14947450880

### ISSUE THREAD

**Issue by sleepy-monax** [AsahiLinux/asahi-installer#35]
Hi, I tried installing Asahi on my MBA.

But the installation failed with the following message:

```
Failed to run process: diskutil quiet addPartition disk0s4 %Linux% %noformat% 14947450880
Output: b'The new partition to be added will be of a human-readable content type of Linux\nThe new partition will be added following disk0s4\n'
```

I followed all the default options so I'm not really sure what I did wrong

Full Install log: https://gist.github.com/sleepy-monax/dd3b85a2e7e886637f50c3245ef7c7bb

**Comment by sleepy-monax** [comment]
Ok running the command by itself seems to yield more information

```
diskutil addPartition disk0s6 %Linux% %noformat% 11947474944     13:08:28
The new partition to be added will be of a human-readable content type of Linux
The new partition will be added following disk0s6
Started partitioning on disk0
Error: -5344: MediaKit reports not enough space on device for requested operation
```

**Comment by sleepy-monax** [comment]
Looks like using 90% instead of max for the OS size let me install it.

**Comment by marcan** [comment]
This is a real problem, it has to do with rounding. A few users have reported it.

I need to sit down and go through the math, which will probably take a day or two since I'm swamped with things... but yes, in the mean time, please use 90% (or even 99%) as the OS size, that should work!

**Comment by marcan** [comment]
Okay, that was dumb. It wasn't rounding, it was just broken code. It wasn't subtracting the size of the stub partition.

Should be fixed now in v0.3.6, now live at alx.sh :)

**Comment by sleepy-monax** [comment]
Thanks for the good work :)
